src/components/peopleImages/peopleImages.tsx

```diff
@@ -0,0 +1,14 @@
+import { IpeopleImages } from '../../type/interfaces';
+export const peopleImages: IpeopleImages = {
+  'no Icon': '/assets/People/noIcon.png',
+  'Luke Skywalker': '/assets/People/Luke_Skywalker.jpg',
+  'C-3PO': '/assets/People/C-3PO.jpg',
+  'R2-D2': '/assets/People/R2-D2.png',
+  'Darth Vader': '/assets/People/Darth_Vader.png',
+  'Leia Organa': '/assets/People/Leia_Organa.jpg',
+  'Owen Lars': '/assets/People/Owen_Lars.jpg',
+  'Beru Whitesun lars': '/assets/People/Beru_Whitesun_lars.jpg',
+  'R5-D4': '/assets/People/R5-D4.webp',
+  'Biggs Darklighter': '/assets/People/Biggs_Darklighter.jpg',
+  'Obi-Wan Kenobi': '/assets/People/Obi-Wan_Kenobi.jpg',
+};
```

Comment:
better to use JS compatible names. They can be used easily like `PeopleImages.Luke`